src/lib/map/analysis/MapSideNavigation.tsx

```diff
@@ -0,0 +1,159 @@
+  // These are for formatting the links to the login
+  // and site URL.
+  display: 'flex',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  fontSize: 15,
+  marginBottom: '1rem',
+};
+
+export function MapSideNavigation({
+  siteInformationProps,
+  children,
+}: MapSideNavigationProps) {
+  const [isExpanded, setIsExpanded] = useState<boolean>(true);
+
+  return (
+    <nav
+      style={{
+        // These styles define the look of the side navigation,
+        // including its width, height, and location on the
+        // page (which depends on the `isExpanded` piece of state.
+        background: menuBackground,
+        // height: 'calc(100% - 150px)',
+        height: '100%',
+        left: isExpanded ? 0 : -200,
+        minHeight: 125,
+        position: 'relative',
+        transition: 'left 0.1s ease',
+        width: 200,
+        marginTop: '2rem',
+        // The parent of this element probably had its pointer-events
+        // set to "none". This restores the default behavior. Without
+        // this, users cannot interact with the side menu via cursor.
+        pointerEvents: 'auto',
+        // Just as with the map header, we need to set a z-index to
+        // ensure that the side menu sits atop the map.
+        zIndex: 10,
+      }}
+    >
+      <button
+        style={{
+          // This makes the button users will click to open/close
+          // the side navigation.
+          background: menuBackground,
+          borderColor: 'transparent',
+          height: 50,
+          width: 50,
+          // These styles pin the expand/collapse to the right of
+          // the lefthand side menu at the nav's vertical center.
+          position: 'absolute',
+          right: -50,
+          top: '50%',
+          transform: 'translate(0%, -50%)',
+          transition: 'all 0.1s ease',
+          // These styles make sure that the button's SVG
+          // is centered on x and y-axis. I think 50% of the
+          // CSS I write is to center things. #tailwind
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+        onClick={() => setIsExpanded((current) => !current)}
+      >
+        <ChevronRight
+          style={{
+            transform: `rotate(${isExpanded ? -180 : 0}deg)`,
+            height: 30,
+            width: 40,
+          }}
+          aria-hidden
+        />
+        <span className="screenReaderOnly">
+          {isExpanded ? 'Close' : 'Open'} {siteInformationProps.siteName} side
+          menu
+        </span>
+      </button>
+      <div
+        style={{
+          // The navigation items are put into a container
+          // which takes up 100% of available space and
+          // allows users to scroll through navigation whenever
+          // there are lots of navigation items and/or the user's
+          // viewport is short.
+          height: '100%',
+          overflowY: 'scroll',
```

Comment:
Is there a reason we want 'scroll' instead of 'auto'? FireFox renders a disabled scrollbar, which looks a bit strange in my opinion.
![image](https://user-images.githubusercontent.com/69446567/223807970-b989c85f-fb7b-4289-a6aa-739c280062ce.png)

Reply:
Oh weird! My FF isn't rendering that way! 😬

![image](https://user-images.githubusercontent.com/24446573/223849436-b6ab9b38-8c6b-458c-8d52-f618c42ab017.png)

Anyways, this is a great callout. I was getting carried away with thinking about having a ton of navigation items. But I think we'll cross that bridge if/when we get there. 

I removed these in https://github.com/VEuPathDB/web-eda/pull/1671/commits/2ffb0860d3a1dd6ffd8cd0f42e17146b91de6dbd

Reply:
@jernestmyers I bet you have an external mouse

Reply:
I do often use an external mouse... 🤔

Reply:
If you just have a trackpad, Mac's will render an overlay scrollbar, but if you have an external mouse, it will render it inline. You can change that behavior, but this is the default.